### DISCUSSION TITLE
Running Selenium on AKS , with Scaled Job as the scaling pattern with existing KEDA

### DISCUSSION THREAD

**Discussion by nikhil-rajan-neu** [SeleniumHQ/docker-selenium#2502] [Q&A] [answered]
This is my value file 
`autoscaling:
  enableWithExistingKEDA: true
  enabled: true
  scaledJobOptions:
    failedJobsHistoryLimit: 0
    jobTargetRef:
      backoffLimit: 0
      completions: 1
      parallelism: 1
    scalingStrategy:
      strategy: default
    successfulJobsHistoryLimit: 0
  scaledOptions:
    maxReplicaCount: 8
    minReplicaCount: 1
    pollingInterval: 10
  scalingType: job
  terminationGracePeriodSeconds: 360
basicAuth:
  enabled: true
  password: xxxxx
  username: xxxxx
chromeNode:
  enabled: true
  replicas: 1
  resources:
    limits:
      cpu: 2
      memory: 4Gi
    requests:
      cpu: 2
      memory: 4Gi
edgeNode:
  enabled: false
firefoxNode:
  enabled: false
global:
  seleniumGrid:
    defaultNodeStartupProbe: httpGet
ingress:
  enabled: false
ingress-nginx:
  enabled: false
tls:
  enabled: false
  ingress:
    enabled: false
`

Selenium Helm chart version is 0.32.0
KEDA : 2.11.2

We are getting the following error during Helm installation 

_Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "clustertriggerauthentications.keda.sh" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "selenium-job": current value is "keda"; annotation validation error: key "meta.helm.sh/release-namespace" must equal "selenium-job": current value is "keda"
 _
 
 Our understanding is that if this is enableWithExistingKEDA: true , is true , then CRD's in keda should not be installed

**Comment by VietND96** [comment, marked as answer]
I saw the problem at your config `autoscaling: enableWithExistingKEDA: true enabled: true`
At a time, only one can be set to `true`, it is `enableWithExistingKEDA` or `enabled`.
Don't set both are true. Because the `autoscaling.enabled` will trigger the installation of sub-chart KEDA.
